projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/ui-tests/tests/convert-job.spec.ts

```diff
@@ -38,6 +39,10 @@ test.describe('convert job', () => {
       .locator('div')
       .filter({ hasText: 'Are you sure you want to convert the Data Job' });
     await page.getByRole('button', { name: 'OK' }).click();
+
+    // Dismiss the dialog that says the operation is running in the background
+    await page.getByRole('button', { name: 'OK' }).click();
+
     await page.locator('div').filter({ hasText: 'Directory not found' });
```

Comment:
@duyguHsnHsn  I'm so confused by this why are we making an assertion that says directory not found when a convert job is successful? 
What does that mean, surely that is an error?

Reply:
@yonitoo @gageorgiev  do you know what the answer to this might be?

Reply:
@murphp15  I cannot really remember, did you try to remove it and test it again?